docs/contributing/installing.md

```diff
@@ -0,0 +1,52 @@
+---
+id: install
+title: Contributing
+sidebar_label: Install
+---
+
+# Install
+
+## Prerequisits
+
+### Setup a development environment
+
+1. Install the latest stable versions of [Git](https://git-scm.com/download).
+1. Configure git user credentials for [every respository](https://help.github.com/articles/setting-your-username-in-git/#setting-your-git-username-for-every-repository-on-your-computer) or [single repository](https://help.github.com/articles/setting-your-username-in-git/#setting-your-git-username-for-a-single-repository)
+1. Install the LTS (Long Term Support) version of [Node](https://nodejs.org/en).
+
+### Setup the source repository
+
+Clone the repository, `cd` into the project, install [Lerna](https://github.com/lerna/lerna) and install dependencies:
+
+``` shell
+git clone https://github.com/Microsoft/fast-dna.git
+cd fast-dna
+npm i --global lerna@3.3.0
+npm i
+```
+
+Install all Lerna dependencies:
+
+``` shell
+lerna bootstrap
+```
+
+Each package in FAST-DNA has its own unique requirements and commands, but the following commands are relatively standard across all packages:
```

Comment:
I left this comment in the other pull request. This section is redundant and should be removed.

Reply:
I removed "Each package in FAST-DNA has its own unique requirements and commands, but the following commands are relatively standard across all packages"